src/pages/Notes.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import Sidebar from "../components/notes/Sidebar";
+import Editor from "../components/notes/Editor";
+import Split from "react-split";
+import { nanoid } from "nanoid";
+
+export default function App() {
+  const [notes, setNotes] = React.useState(
+    () => JSON.parse(localStorage.getItem("notes")) || []
+  );
+  const [currentNoteId, setCurrentNoteId] = React.useState(
```

Comment:
useState is exported from react
```suggestion
  const [currentNoteId, setCurrentNoteId] = useState(
```